ruby_version_file_parser_test.go

```diff
@@ -0,0 +1,99 @@
+package bundleinstall_test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	bundleinstall "github.com/paketo-buildpacks/bundle-install"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+const RUBY_VERSION_FILE_TEMPLATE = "%s\n"
+
+func testRubyVersionFileParser(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		path   string
+		parser bundleinstall.RubyVersionFileParser
+	)
+
+	it.Before(func() {
+		file, err := os.CreateTemp("", ".ruby-version")
+		Expect(err).NotTo(HaveOccurred())
+		defer file.Close()
+
+		path = file.Name()
+
+		parser = bundleinstall.NewRubyVersionFileParser()
+	})
+
+	it.After(func() {
+		Expect(os.RemoveAll(path)).To(Succeed())
+	})
+
+	context("ParseVersion", func() {
+		context("when given different types of versions", func() {
+			it("parses the versions correctly", func() {
+				versions := []string{
+					"2",
+					"2.6",
+					"2.6.0",
+					"2.7.0",
+					"2.10.0",
+					"2.10.10",
+					"10.0.0",
+				}
+
+				for _, v := range versions {
+					Expect(os.WriteFile(path, []byte(fmt.Sprintf(RUBY_VERSION_FILE_TEMPLATE, v)), 0644)).To(Succeed())
```

Comment:
Sorry for the nitpick, but for the test setup, can you add this to an `it.Before()` block? It clarifies which bit of the test is set up, and which bit is testing the actual functionality.

Reply:
No problem, I moved it to an `it.Before()` block.